.github/commenter.yml

```
@@ -0,0 +1,61 @@
+Prelude:
+  where:
+    path:
+      matches: "library/Freckle/App/Prelude.hs"
+  body: |
+    :wave: Hi, I see you made a change to Prelude! If you're adding a new export
+    please keep the following guidelines in mind.
+
+    **Ubiquity**
+
+    We shouldn't be adding exports to the prelude that aren't commonly used
+    throughout our codebase, even if we feel they _might_ be more commonly used
+    in the future.
+
+    Good rationale:
+
+    > I'm adding `void` because we use it everywhere in our app code and tests
+
+    Bad rationale:
+
+    > I just found the `.:?-` combinator and think we might want to use it in `X`
+
+    **Compatibility**
+
+    We shouldn't add exports that conflict with other commonly-used functions.
+
+    For example, [`Data.Function (on)`][func-on] was intentionally excluded from
+    this prelude because it conflicts with [`Database.Esqueleto (on)`][esq-on]
+    which we often import.
+
+    **Idiomatic**
+
+    We'd like to keep the codebase accessible to Frecklers and, more broadly,
+    Haskellers. We're open to trying new things but, until they've been proven
+    out a bit, consistency is key!
+
+    This concept can be a little squishy but try to think along the lines of
+
+    > If I learned Haskell by contributing to the Freckle codebase, am I likely
+      to be comfortable with the pattern(s) this export encourages?
```

If you're hard-breaking generally (which I like), I think this should be like this:
```suggestion
    > to be comfortable with the pattern(s) this export encourages?
```

And ditto L45.